src/wallet/account.ts

```diff
@@ -170,5 +166,15 @@ export class WalletAccount extends Account implements AccountInterface {
     return signMessage(this.walletProvider, typedData);
   }
 
+  static async connect(
+    provider: ProviderInterface,
+    walletProvider: StarknetWalletProvider,
+    cairoVersion?: CairoVersion,
+    silentMode: boolean = false
```

Comment:
I like my implementation better. Separating "connect" and "connectSilent" is a better approach imo, then passing a boolean here

Reply:
OK will add connectSilent as alias on fixed connect, I think these best compromise on this.

Reply:
c83e304daa58d97b0e4abac6c06f03fb3ba1652e